blueprints/dumb/files/__test__/components/__name__.spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import <%= pascalEntityName %> from 'components/<%= pascalEntityName %>/<%= pascalEntityName %>'
```

Comment:
> it would be nice to selectively wrap files in folders, since I don't really want ducks/containers/etc. > to be wrapped in an extra folder, only components and views.

just to confirm: you can selectively wrap different blueprints in folders.  Initially I had that as a setting for the CLI but I think using the blueprints model that ember-cli came up with is far superior since it lets each user customize their blueprints and file structures on their own